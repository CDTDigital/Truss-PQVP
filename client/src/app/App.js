import React, { PropTypes } from 'react';
import { Header } from '../header/Header';

export const App = ({ children }) => (
  <div>
    <Header />
    <div>
      {children}
    </div>
  </div>
);

export default App;

App.propTypes = {
  children: PropTypes.element.isRequired,
};
