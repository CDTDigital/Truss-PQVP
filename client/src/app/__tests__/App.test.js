import React from 'react';
import App from '../App';
import renderer from 'react-test-renderer';

test('App renders', () => {
  fetchMock.get('*', {text: 'hello, PQVP!'});

  const component = renderer.create(
    <App />
  );

  let tree = component.toJSON();
  expect(tree).toBeDefined();
});
