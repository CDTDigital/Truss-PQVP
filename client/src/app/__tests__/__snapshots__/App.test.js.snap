exports[`test App renders 1`] = `
<div>
  <div>
    <h1>
      The Emergency Alert App Header
    </h1>
  </div>
  <div />
</div>
`;
